src/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -62,6 +64,36 @@ public WatchProtocolDecoder(Protocol protocol) {
             .expression("(.*)")                  // cell and wifi
             .compile();
 
+    private static final String STATE = "State";
+    private static final String ALARM = "Alarm";
```

Comment:
What is this for?

Reply:
The terminal statement | 00000000 | Represents 			in Hexadecimal string,and the meaning isas following:The high 			16bit is alarming,the low 16 bit is statement. 			Bit(from 			0)        Meaning(1 			effective) 			 			0                  Low-battery 			 			1                  Out of fence  			 			 			2                  In fence 			 			3                  Bracelet statement 			 			16                 SOS alarming 			 			17                 Low-battery alarming 			 			18                 Out of fence alarming 			 			19                 In fence alarming 			20     			            Watch-take-off alarming
-- | -- | --



from here: https://dl.dropboxusercontent.com/s/2hid9vaa2vno4jr/Communication%20Protocol.doc  (linked from https://www.traccar.org/protocols/)

Reply:
Status should be ignored, unless we have special attribute for it. Alarm should only include alarms.

Reply:
So the current code in master is wrong.
Either new constants could be added to Position to accomodate status,
or we can remove bits 0-3. from the above code.
I still think that we should allow multiple alerts at the same time, it's possible to have sos, either entering or exiting geofence (actually it could be both at the same time, depends on how the user draw them:) and watch removed alert at the same time.

Reply:
It's very possible that something is not decoded correctly.

Yes, it would be good to support multiple alerts. There was even some work done for it somewhere.

Reply:
ok, so that part of the patch is ok at least:)
So do you want me to remove STATE, ALARM Strings and bits 0-3 from the Map, leaving only the alerts? Or should we somehow add/keep the information about the status? Either by renaming the method to decodeStatus (I think this is better, and BTW we could add then motion as well) or if you prefer to have an alert only method, then by adding another one for the "rest" of the bits: decodeSatus() ?

Reply:
I would prefer to separate two.